src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3815,6 +3818,80 @@
     public static final int TEXT_BLOCK_LITERAL_END =
             GeneratedJavaTokenTypes.TEXT_BLOCK_LITERAL_END;
 
+    /**
+     * The {@code yield} keyword.  This element appears
+     * as part of a yield statement.
```

Comment:
we need AST example to show context of usage.
as `yield` is very specific "keyword" that is activated in switch only, in comparison to other keywords that do not allow to create any identifier with same name.

Reply:
Fixed.

Reply:
Fixed.